app/_locales/cs/messages.json

```diff
@@ -1,10 +1,4 @@
 {
-  "privacyMode": {
```

Comment:
Many locales appear to still have these two messages (e.g. `ja`, `ko`, `nl`)

Reply:
Good catch, removed the rest of 'em (I seemingly stopped half way through)